### PR TITLE
Create docker image for kubelet

### PR DESCRIPTION
**Pull request by pmorie**
@vmarmol Not sure who else to tag on this one.

### CODE REVIEW

**Comment by pmorie** [comment]
A couple things I found about the wheezy backport used by the hypercube image:

1.  Not the right version of libc required for kubelet
2.  No nsenter binary even though util-linux is installed

So, I used centos for now; I'm completely open to suggestions about a better image to use.

File: cluster/images/kubelet/.gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+kubelet
```
Comment (vmarmol) [inline cluster/images/kubelet/.gitignore]: I think we tend not to add `.gitignore` files :P

File: cluster/images/kubelet/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,9 @@
+FROM centos
```
Comment (vmarmol) [inline cluster/images/kubelet/Dockerfile]: Maybe do something similar to what we do for nginx:

https://github.com/GoogleCloudPlatform/kubernetes/blob/master/cluster/images/nginx/Dockerfile

**Comment by vmarmol** [comment]
LGTM, nothing really a blocker.

**Comment by dchen1107** [comment]
How are we going to handle development version kubelet image?

**Comment by pmorie** [comment]
@dchen1107 For that, there's a 'local' target in the makefile.  There are a ton of ways we could automate it a bit better -- any preference?

File: cluster/images/kubelet/Dockerfile (hunk 2 above)
Comment (pmorie) [inline cluster/images/kubelet/Dockerfile]: Is there a way to verify whether util-linux via this list would contain nsenter?

Comment (vmarmol) [inline cluster/images/kubelet/Dockerfile]: Outside of trying it out I'm not sure...

Comment (pmorie) [inline cluster/images/kubelet/Dockerfile]: Also need to we can get the right libc:

```
/kubelet: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.14' not found (required by /kubelet)
```

Comment (pmorie) [inline cluster/images/kubelet/Dockerfile]: For the record on the wheezy image the nginx image is based on:

```shell
root@d232b986ba56:/# ldd --version
ldd (Debian EGLIBC 2.13-38+deb7u7) 2.13
```

Comment (vmarmol) [inline cluster/images/kubelet/Dockerfile]: Aaah nvm, let's just use centos

**Comment by pmorie** [comment]
Ok, I'm going to remove the gitignore file.  Let's make a follow-up issue to make the image uniform with the other 'official' images.  To me it would be ideal to just be able to build static binaries and dump them into a scratch image, but there are a couple packages in the kubelet that need native code.

**Comment by pmorie** [comment]
Aight, should be ready to go in, assuming everyone is cool w/ redirect to host logfile as a follow-up.

**Comment by vmarmol** [comment]
LGTM, will merge tomorrow during merge hours.

**Pull request merged**